File: aspnet-mvc-demo/AspnetMvcDemo/Controllers/Pages/TicketsController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace AspnetMvcDemo.Controllers.Pages
{
    public class TicketsController : Controller
    {

         public ActionResult Edit()
        {
            return View("~/Views/Pages/Tickets/Edit.cshtml");
        }

         public ActionResult List()
        {
            return View("~/Views/Pages/Tickets/List.cshtml");
        }

    }
}

File: laravel-demo/app/Http/Controllers/Pages/TicketsController.php
<?php

namespace App\Http\Controllers\Pages;

use App\User;
use App\Http\Controllers\Controller;

class TicketsController extends Controller
{
    public function edit()
    {
        return view('pages.tickets.edit', ['title' => 'Edit ticket - Pages']);
    }

    public function list()
    {
        return view('pages.tickets.list', ['title' => 'Ticket list - Pages']);
    }

}
